disputer/disputer.js

```diff
@@ -128,23 +128,26 @@ class Disputer {
       const dispute = this.empContract.methods.dispute(disputeableLiquidation.id, disputeableLiquidation.sponsor);
 
       // Simple version of inventory management: simulate the transaction and assume that if it fails, the caller didn't have enough collateral.
+      let gasLimit;
       try {
-        await dispute.call({ from: this.account, gasPrice: this.gasEstimator.getCurrentFastPrice() });
+        await dispute.call({ from: this.account });
+        gasLimit = Math.floor((await dispute.estimateGas({ from: this.account })) * 1.5);
```

Comment:
Do you think we really need the 1.5 buffer? If so, maybe define this in the config or as a constant above the file

Reply:
I think truffle uses `1.25` by default: https://github.com/trufflesuite/truffle/issues/2539. Not sure of all the details around why that's necessary, but I've always been unclear if it's estimating the gas _required_ or the gas _used_ (since the used gas may be lower than the gas that you must send to get a successful txn).

Reply:
I do think we need the buffer. Often the gas used by the tx is different from what the estimation provides. It could be a config but I dont think there are any situations in which the user should be messing with this. We could set it to 1.25 like truffle does for consistency and then leave it hardcoded in the script.

Reply:
Makes sense. I think you should set the 1.25x multiple early as a constant and then re-use that so you and future devs don't fat finger any multiples